src/components/links/LinkBase.tsx

```diff
@@ -22,8 +22,8 @@ import styled from '@emotion/styled';
 import React, { HTMLAttributeAnchorTarget } from 'react';
 import { useIntl } from 'react-intl';
 import { Link as RouterLink, LinkProps as RouterLinkProps } from 'react-router-dom';
-import { isSonarLink } from '../../common/helpers/url';
-import { OpenNewTabIcon } from '../../common/icons/OpenNewTabIcon';
+import { isSonarLink } from '~common/helpers/url';
+import { IconLinkExternal } from '../icons';
```

Comment:
Here it's okay (icons are low-level), but it's better to import straight from the module rather than from an index:
otherwise, it can lead to circular dependencies